frontend/public/components/utils/kebab.tsx

```diff
@@ -609,13 +617,21 @@ Kebab.factory = kebabFactory;
 Kebab.columnClass = 'pf-v6-c-table__action';
 Kebab.getExtensionsActionsForKind = getExtensionsKebabActionsForKind;
 
+type CtaModal<P = any> = {
```

Comment:
Nit - we should reuse the same `CtaModal` type instead of having multiple declarations.